README.md

```diff
@@ -8,23 +8,13 @@ Before getting started, you'll need an account that has CMS Hub Enterprise and a
 
 1. Clone this repo to your machine
 2. Install dependencies via `npm install`
-3. Create a `hubspot.config.yml` file as follows:
-
-```yaml
-defaultPortal: demo
-portals:
-  - name: demo
-    portalId: <portalId>
-    authType: apikey
-    apiKey: <apiKey>
-```
-
+3. If you already have a `hubspot.config.yml` file set up, run `hs auth` to authenticate with your account. Otherwise, run `hs init` to set up your config file.
 4. Run `npm run setup` to create the custom object
-5. Run `npm run upload` to upload the `hub-homes` theme
+5. Run `npm run upload` to upload the `Hub Homes` theme
 6. Add your API key by running `hs secrets add APIKEY <apikey>`
```

Comment:
Could we make this more clear that what you are doing here is setting up the secret for a serverless function?

Reply:
Added